### PR TITLE
localization changes

### PR DESCRIPTION
**Pull request by nipunja**

### CODE REVIEW

File: Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1 [hunk 1]
```diff
@@ -599,7 +599,7 @@ function InvokeTestAgentConfigExe([string[]] $Arguments, [string] $Version, [Sys
         return $out
     }
 
-    throw "Did not find TestAgentConfig.exe at : $exePath. Ensure that TestAgent is installed."
+    throw (Get-LocalizedString -Key "Did not find TestAgentConfig.exe at : '{0}'. Ensure that TestAgent is installed." -ArgumentList $exePath)
```
Comment (acesiddhu) [inline Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1]: are these string in resx? also this is not directly run by task. but on test agent

**Comment by kmkumaran** [comment]
Looks for the deployment tasks.

File: Tasks/Docker/Docker-Deployment.ps1 [hunk 2]
```diff
@@ -170,13 +172,13 @@ function Handle-ImageNotFoundError($response)
     $imageNotFoundErrormsg = ("image {0} not found" -f $repository)
     if($response.contains($imageNotFoundErrormsg))
     {
-        throw ("{0} on docker hub" -f $imageNotFoundErrormsg)
+        throw (Get-LocalizedString -Key "image '{0}' not found on docker hub" -ArgumentList $repository)
```
Comment (roniga) [inline Tasks/Docker/Docker-Deployment.ps1]: Image

File: Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1 [hunk 3]
```diff
@@ -155,13 +155,13 @@ function Get-Resources
...
         {
-            throw "Unable to get FQDN for all resources in ResourceGroup : $resourceGroupName"
+            throw (Get-LocalizedString -Key "Unable to get FQDN for all resources in ResourceGroup : '{0}'" -ArgumentList $resourceGroupName)
         }
         else
         {
             if($fqdnErrorCount -gt 0 -and $fqdnErrorCount -ne $azureResourceGroupResources.Count)
             {
-                 Write-Warning "Unable to get FQDN for $fqdnErrorCount resources in ResourceGroup : $resourceGroupName" -Verbose
+                Write-Warning (Get-LocalizedString -Key "Unable to get FQDN for {0} resources in ResourceGroup : '{1}'" -ArgumentList $fqdnErrorCount, $resourceGroupName)
```
Comment (roniga) [inline Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1]: Add quotes, '{0}'

**Pull request merged**